### PR TITLE
Removes unused dependency on base64

### PR DESCRIPTION
**Pull request by TheGreyDiamond**
This removes the dependency on base64

### CODE REVIEW

**Comment by klaernie** [comment]
Could you please take a look again, there were some changes to the general coding conventions, which now produces a confusing diff.

File: node_helper.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
-const fs = require("fs");
-const path = require("path");
-const validUrl = require("valid-url");
+const fs = require("fs")
+const path = require("path")
+const validUrl = require("valid-url")
+const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
```
Comment (KristjanESPERANTO) [inline node_helper.js:4]: fetch, moment, ICAL, IcalExpander now are two times here.

base64 we can be removed.

Comment (TheGreyDiamond) [reply node_helper.js]: Yes, was a merge problem. If fixed now!

**Comment by klaernie** [comment]
I decided to rebase this change myself. This keeps the spirit of the request, but also incorporates the changes made in the meantime on our end.

**Pull request closed without merging**